src/zotonic_sup.erl

```diff
@@ -248,3 +258,11 @@ get_extensions() ->
           permanent, 5000, worker, dynamic}
      end
      || F <- Files].
+
+%% @doc Let sites return their own keys and certificates.
+sni_fun(ServerName) ->
+    case z_sites_dispatcher:get_host_for_domain(ServerName) of
+        undefined -> undefined;
+        {ok, Host} -> 
+            z_notifier:first(#ssl_options{server_name=ServerName}, z_context:new(Host))
```

Comment:
Missen that one. I remember looking in the notification rst. Adding it later today. 

I want to move this to the stable branch later on. 

Verstuurd vanaf mijn iPhone

> Op 11 mei 2016 om 07:31 heeft David de Boer <notifications@github.com> het volgende geschreven:
> 
> In src/zotonic_sup.erl:
> 
> > @@ -248,3 +258,11 @@ get_extensions() ->
> >            permanent, 5000, worker, dynamic}
> >       end
> >       || F <- Files].
> > +
> > +%% @doc Let sites return their own keys and certificates.
> > +sni_fun(ServerName) ->
> > +    case z_sites_dispatcher:get_host_for_domain(ServerName) of
> > +        undefined -> undefined;
> > +        {ok, Host} -> 
> > +            z_notifier:first(#ssl_options{server_name=ServerName}, z_context:new(Host))
> This is a new notification that should be documented. Can you add it to the SSL module’s RST?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>